jax/_src/config.py

```diff
@@ -490,12 +490,20 @@ def update_thread_local_jit_state(**kw):
           'DEBUG.'))
 
 distributed_debug = config.define_bool_state(
-    name="jax_distributed_debug",
+    name='jax_distributed_debug',
     default=False,
     help=('Enable logging useful for debugging multi-process distributed '
           'computations. Logging is performed with `absl.logging` at WARNING '
           'level.'))
 
+hlo_metadata_source_file_prefix_removal_regex = config.define_string_state(
+    name='jax_hlo_metadata_source_file_prefix_removal_regex',
+    default=None,
+    help=('Used to strip a prefix from the source_path metadata of HLO '
+          'instructions. If set, will be searched as a regex in each '
+          'source_file, and if found, everything up to and including the '
```

Comment:
Perhaps it would be cleaner to remove *exactly* things that match? If you want to remove up to the match prefix it with `.*`.

Reply:
I changed to re.sub() as suggested, which effective does this.